site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -45,6 +46,29 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const editThread = (title) => {
+    cy.get('[data-testid="thread-list-item"]').contains(title).click();
+    cy.get('[data-testid="create-post-head"]').should('contain', title);
+    cy.get('[data-testid="edit-post-button"]').first().click();
+    cy.get('[data-testid="upload1"]').find('[for="input-file1"]').selectFile('cypress/fixtures/sea_animals.png');
+    cy.get('[data-testid="file-upload-table-1"]').find('.file-label').should('contain', 'sea_animals.png');
+    cy.get('[data-testid="file-upload-table-1"]').click();
+    cy.get('[data-testid="file-upload-table-1"]').should('not.contain');
```

Comment:
What is the point of this line?

Reply:
I removed that part and realized that I was using the same code. I implemented it for the delete button which comes when we upload any attachment [ as shown in image] but it is not working, we can do it after uploading the image and then delete it.

<img width="1792" alt="Screenshot 2024-05-08 at 8 05 50 PM" src="https://github.com/Submitty/Submitty/assets/119070053/d59392e8-f44f-411c-9ef0-82b32610ee9c">